include/SQLfunctions.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+function GetABlogpost($blogpostId){
```

Comment:
This file looks fine, but why is it structured differently from your other functions? Seems like maybe this file should be `blogFunctions.php` and you might want some other include files for other functions.